src/navigation/examples/vertical-navigation-basic.js

```diff
@@ -392,9 +394,19 @@
            href: "#/adipscing"
         },
         {
-           title: "Lorem",
-           iconClass: "fa fa-gamepad",
-           href: "#/lorem"
+           title: "Help",
+           iconClass: "fa pficon-help",
+           href: "#/help",
+           mobileOnly: "true"
+        },
```

Comment:
This should have Help and About as children to be consistent with the header.

Reply:
These changes should be made in the `-router` example as well

Reply:
> This should have Help and About as children to be consistent with the header.

Wasn't crazy about having sub-menus:
```
Help -> Help
        About
```
So left out 'About' as maybe something you wouldn't need at mobile.

Reply:
OK, just thinking we should show consistency.